OpenRA.Mods.Common/Widgets/Logic/MapChooserLogic.cs

```diff
@@ -43,7 +43,12 @@ public class MapChooserLogic : ChromeLogic
 			this.modData = modData;
 			this.onSelect = onSelect;
 
-			var approving = new Action(() => { Ui.CloseWindow(); onSelect(selectedUid); });
+			var approving = new Action(() =>
+			{
+				Ui.CloseWindow();
+				if (onSelect != null)
```

Comment:
This can simplify to remove the if using `onSelect?.Invoke(selectedUid);`

Reply:
Changed and tested, ?.Invoke worked whether it's null(It's not null when user is the host) or not.